Our LibreSSL requires newer openssl-sys.
Also, older num-bigint appears to be broken on ARM.

Some builds also fail with unused import.

--- fractal-gtk/src/app/backend_loop.rs
+++ fractal-gtk/src/app/backend_loop.rs
@@ -12,8 +12,6 @@ use std::thread;
 use crate::backend::BKResponse;
 use fractal_api::error::Error;
 
-use std::sync::mpsc::RecvError;
-
 pub fn backend_loop(rx: Receiver<BKResponse>) {
     thread::spawn(move || {
         while let Ok(recv) = rx.recv() {
--- scripts/cargo.sh
+++ scripts/cargo.sh
@@ -5,6 +5,9 @@ export CARGO_TARGET_DIR="$3"/target
 export CARGO_HOME="$CARGO_TARGET_DIR"/cargo-home
 export FRACTAL_PROFILE="$4"
 
+cargo update --manifest-path $1/Cargo.toml -p fractal-gtk --package openssl-sys --precise 0.9.58
+cargo update --manifest-path $1/Cargo.toml -p fractal-gtk --package num-bigint --precise 0.2.3
+
 if [ "$FRACTAL_PROFILE" = "Devel" ]
 then
     echo "DEBUG MODE"
